docs/linux/sql-server-linux-active-directory-authentication.md

```diff
@@ -96,19 +89,12 @@ Use the following steps to join a [!INCLUDE[ssNoVersion](../includes/ssnoversion
 
    - **RHEL**:
 
-     Edit the `/etc/sysconfig/network-scripts/ifcfg-eth0` file (or other interface config file as appropriate) so that your AD domain controller's IP address is listed as a DNS server:
-
-     ```/etc/sysconfig/network-scripts/ifcfg-eth0
-     <...>
-     PEERDNS=no
-     DNS1=**<AD domain controller IP address>**
-     ```
-
-     After editing this file, restart the network service:
-
-     ```bash
-     sudo systemctl restart network
-     ```
+     Insure there are no DNS1 DNS2 or PEERDNS entries specified in `/etc/sysconfig/network-scripts/ifcfg-eth*` files
+     After editing this file, restart the network service (eth0 specified in example):
+   
+      ```bash
+      sudo ifdown eth0 && sudo ifup eth0
```

Comment:
Why are we changing the command here?

Reply:
Because it's usually not necessary to restart all of networking.

Reply:
We restart networking to regenerate resolv.conf